tools/image_processing/fractal/macros.xml

```diff
@@ -0,0 +1,31 @@
+<macros>
+    <token name="@FRACTAL_VERSION@">1.4.3</token>
```

Comment:
use TOOL_VERSION https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions if you want to have automatic bot version bumps